crates/swift-bridge-ir/src/bridged_type/.bridged_opaque_type.rs.rustfmt

```diff
@@ -0,0 +1,764 @@
+use crate::bridged_type::{
```

Comment:
Need to delete this file

Reply:
oh shoot! Sorry about that, thought I avoided committing that one. Done